--- fvs-0.0.9/gfvs/src/Makefile.am.orig	Sat May 10 18:47:48 2003
+++ fvs-0.0.9/gfvs/src/Makefile.am	Thu Jun 26 12:52:50 2003
@@ -3,6 +3,7 @@
 INCLUDES = \
 	-DPACKAGE_DATA_DIR=\""$(datadir)"\" \
 	-DPACKAGE_LOCALE_DIR=\""$(prefix)/$(DATADIRNAME)/locale"\" \
+	-I. \
 	@PACKAGE_CFLAGS@
 
 bin_PROGRAMS = gfvs
@@ -13,6 +14,6 @@
 	interface.c interface.h \
 	callbacks.c callbacks.h
 
-gfvs_LDADD = @PACKAGE_LIBS@ $(INTLLIBS)
+gfvs_LDADD = @PACKAGE_LIBS@ $(INTLLIBS) ../../src/libfvs.la
 
 CLEANFILES = *~ *.bak
--- fvs-0.0.9/gfvs/src/callbacks.c.orig	Sat Apr 19 17:43:25 2003
+++ fvs-0.0.9/gfvs/src/callbacks.c	Thu Jun 26 13:05:20 2003
@@ -48,11 +48,11 @@
   gtk_main();
   if (filename==NULL)
   {
-    FvsFile_t* f = NULL;
-    FileCreate(&f);
-    ImageImport(f, filename, FILE_READ);
+    FvsFile_t f;
+    f = FileCreate();
+    FileOpen(f, filename, FvsFileRead);
 
-    FileDestroy(&f);
+    FileDestroy(f);
   }  
 }
 
--- fvs-0.0.9/gfvs/configure.in.orig	Wed Mar 26 22:03:54 2003
+++ fvs-0.0.9/gfvs/configure.in	Thu Jun 26 13:12:26 2003
@@ -7,6 +7,7 @@
 
 AC_ISC_POSIX
 AC_PROG_CC
+AC_PROG_LIBTOOL
 AM_PROG_CC_STDC
 AC_HEADER_STDC
 
